### DISCUSSION TITLE
Filepicker gives a COMException error when used.

### DISCUSSION THREAD

**Discussion by kruddick** [dotnet/maui#20755] [Q&A]
I am using a file picker in my Maui project. I have tried to figure out the cause but can't seem to figure it out. I have created small test projects to test the issue in isolation and it seems to work fine. I have used the same code in both the regular application and the test applications.

Here is my UI. Click browse produces the error listed on the bottom.
![image](https://github.com/dotnet/maui/assets/19706520/9849afae-30e2-4bb4-a931-7a0b80d83213)

the button leads to this code block: 
![image](https://github.com/dotnet/maui/assets/19706520/ed60dcba-340f-4361-824f-984c782390a5)

that leads to this code block:
![image](https://github.com/dotnet/maui/assets/19706520/86109f03-7426-4bbf-af39-359117e2d1b8)

I keep thinking that maybe it has something to do with not using the packaging and having 

`<WindowsPackageType>None</WindowsPackageType>`
in my project file.

From the stack trace it seems to be throwing in the filepicker code and not necessarily my code. I would appreciate any tips to get this to work. I have though about trying to call the native filepicker in another way but haven't been able to figure out how to do that either.

Thanks.

**Comment by kruddick** [comment]
I think I found my answer. This is a really old bug where File Dialog will not work with admin permissions. Need to rethink using MAUI for this project. I REALLY wish someone would update the documentation just to say this is known issue.

**Reply by daniel-hemmen** [reply]
Wow so happy I stumbled upon this, I thought I was going crazy. I open Visual Studio as admin by default and could not get the FolderPicker to work. Opening Visual Studio regularly fixed it.
